indico/modules/events/contributions/forms.py

```diff
@@ -187,6 +188,9 @@ class ContributionTypeForm(IndicoForm):
     """Form to create or edit a ContributionType"""
 
     name = StringField(_("Name"), [DataRequired()])
+    is_private = BooleanField(_("Private contribution type"), widget=SwitchWidget(), default=False,
+                              description=_("If unchecked, the contribution type can be assigned by managers/judges "
```

Comment:
"If selected, this contribution type cannot be selected by users submitting an abstract."?

Reply:
"selected" twice sounds a bit weird. Maybe ".. cannot be chosen by users...?